soc/arm/microchip_mec/mec172x/reg/mec_regaccess.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 Microchip Technology Inc. and its subsidiaries.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _MEC_REGACCESS_H
+#define _MEC_REGACCESS_H
+
+#include <stdint.h>
+
+#define MMCR32(a)	*((volatile uint32_t *)(uintptr_t)(a))
```

Comment:
again we have a zephyr way of doing this, can we not use those?

Reply:
Agreed once I find the zephyr versions.

Reply:
Removed mec_defs.h and mec_regacces.h  Using zephyr inline routines in sys_io.h